specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2023-03-01-preview/examples/Caches_CreateOrUpdate.json

```diff
@@ -0,0 +1,327 @@
+          "lastModifiedByType": "User",
+          "lastModifiedAt": "2020-01-02T17:18:19.1234567Z"
+        },
+        "properties": {
+          "cacheSizeGB": 3072,
+          "scalingFactor": 1.0,
+          "health": {
+            "state": "Transitioning",
+            "statusDescription": "Cache is being created.",
+            "conditions": [
+              {
+                "timestamp": "2021-04-21T18:25:43.511Z",
+                "message": "Cannot contact DNS server"
+              }
+            ]
+          },
+          "mountAddresses": [
+            "192.168.1.1",
+            "192.168.1.2"
+          ],
+          "provisioningState": "Succeeded",
+          "subnet": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.Network/virtualNetworks/scvnet/subnets/sub1",
+          "upgradeStatus": {
+            "currentFirmwareVersion": "5.3.23",
+            "firmwareUpdateStatus": "available",
+            "firmwareUpdateDeadline": "2019-04-21T18:25:43.511Z",
+            "lastFirmwareUpdate": "2019-01-21T18:25:43.511Z",
+            "pendingFirmwareVersion": "5.3.24"
+          },
+          "networkSettings": {
+            "mtu": 1500,
+            "dnsServers": [
+              "10.1.22.33",
+              "10.1.12.33"
+            ],
+            "dnsSearchDomain": "contoso.com",
+            "ntpServer": "time.contoso.com"
+          },
+          "securitySettings": {
+            "accessPolicies": [
+              {
+                "name": "default",
+                "accessRules": [
+                  {
+                    "scope": "default",
+                    "access": "rw",
+                    "suid": false,
+                    "submountAccess": true,
+                    "rootSquash": false
+                  }
+                ]
+              }
+            ]
+          },
+          "upgradeSettings": {
+            "upgradeScheduleEnabled": true,
+            "scheduledTime": "2022-04-26T18:25:43.511Z"
+          },
+          "encryptionSettings": {
+            "keyEncryptionKey": {
+              "keyUrl": "https://keyvault-cmk.vault.azure.net/keys/key2048/test",
+              "sourceVault": {
+                "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/scgroup/providers/Microsoft.KeyVault/vaults/keyvault-cmk"
+              }
+            }
+          },
+          "directoryServicesSettings": {
+            "activeDirectory": {
+              "primaryDnsIpAddress": "192.0.2.10",
+              "secondaryDnsIpAddress": "192.0.2.11",
+              "domainName": "contosoAd.contoso.local",
+              "domainNetBiosName": "contosoAd",
+              "cacheNetBiosName": "contosoSmb",
+              "domainJoined": "No"
+            },
+            "usernameDownload": {
+              "extendedGroups": true,
+              "usernameSource": "LDAP",
+              "groupFileURI": "",
+              "userFileURI": "",
```

Comment:
Shouldn't your RP reject invalid URIs? So empty strings shouldn't be a valid example.

Reply:
Fixed the example and provided values.